documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -2499,6 +2501,12 @@ For example, in an organization with archive destinations `LOG_ARCHIVE_DEST_2` a
 }
 ----
 
+[NOTE]
+====
+Do not set `log.mining.archive.destination.name` to the path where the archive logs are written by Oracle.
+Instead, this property should refer to the value in the `DEST_NAME` column in the `V$ARCHIVE_DEST_STATUS` table for the preferred row that satisfies your archive log retention policy.
```

Comment:
```suggestion
Set the property to the name of the archive log destination in the `DEST_NAME` column for a row in the `V$ARCHIVE_DEST_STATUS` table that satisfies your archive log retention policy.
```
@Naros I'm not sure what the criteria for preferred might be in the original description.  I'm inclined to omit it, as I did in my edit. Is there some guidance that we could offer toward interpreting that qualifier, or is it non-essential?

Reply:
That seems reasonable to drop it; it reads fine to me without it, too.